src/app/components/vm-main/vm-main.component.ts

```diff
@@ -71,16 +84,41 @@ export class VmMainComponent implements OnInit, OnDestroy {
       });
 
     this.readOnly$ = this.vmService.GetReadOnly(this.viewId);
+
+    this.currentUser$ = this.authService.user$.pipe(
+      switchMap((u) => {
+        return this.userService.getUser(u.profile.sub);
+      })
+    );
+
+    this.loggingEnabled$ = this.vmUsageLoggingSessionService.getIsLoggingEnabled();
+
+    this.teamService.GetAllMyTeams(this.viewId ).pipe(take(1)).subscribe(tms => {
```

Comment:
Might be easier/more efficient to use the /permissions endpoint to get the user's permissions by viewId in the future